### ISSUE TITLE
FSE: Remove "Nav Menus" panel from Customizer

### ISSUE THREAD

**Issue by jffng** [Automattic/wp-calypso#62501]
### What

For FSE enabled themes, remove the Menus panel from the Customizer:

From | To
--- | ---
<img width="486" alt="Screen Shot 2022-04-04 at 3 56 27 PM" src="https://user-images.githubusercontent.com/5375500/161621749-aa08b621-600f-4c36-be96-20d3ec164956.png"> | <img width="488" alt="Screen Shot 2022-04-04 at 3 56 07 PM" src="https://user-images.githubusercontent.com/5375500/161621759-46404185-1d0d-4b81-8840-d732ee64f17b.png">

### Why

https://github.com/Automattic/wp-calypso/pull/60692 already removed Appearance > Menus for these themes. I think it would be more consistent to also remove this Menus editing view from the Customizer so that the Site Editor is the single source of truth for menus, otherwise there is the possibility that they become out of sync. More context:

https://github.com/Automattic/themes/issues/5628

**Comment by jffng** [comment]
cc @Addison-Stavlo in case you have thoughts here :)

**Comment by liviopv** [comment]
@jffng I'm working on a P2 post that will touch on this, and I'm wondering Customizer > Menus load for some themes and not on others.

If I install a WordPress.com Premium theme like Kerr, I get access to menus from the customizer, but when I select TT3, there's no path to edit menus in the customizer, even if the site has classic themes previously saved.

I'm unfamiliar with the code, but I think this could be theme-dependent?

**Comment by jffng** [comment]
@liviopv yes it's indeed theme dependent — if the site is a [block theme](https://developer.wordpress.org/block-editor/how-to-guides/themes/block-theme-overview), menus should only editable via the site editor.

That said, I think this issue can now be closed.

**Comment by liviopv** [comment]
Reopening as this is still a pain point for Happiness

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept

**Comment by dsas** [comment]
Will be resolved by https://github.com/Automattic/wp-calypso/issues/71130
